src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1137,10 +1137,8 @@ static void HandleScanInfo(Eventinfo *lf,int *socket,cJSON *event) {
 
     switch (result_db)
     {
-        case -1:
-            merror("Error querying policy monitoring database for agent %s", lf->agent_id);
-            break;
         case 0:
+        case 1:
```

Comment:
Case 1 is a bit different than case 0, in case 1 we have not `wdb_response` since we know the DB is empty. Let's ask for the scan directly and show a descriptive log message.